public/data/processes.json

```diff
@@ -7015,7 +7015,7 @@
     },
     "heat_MJ": 0.0,
     "elec_pppm": 0.0,
-    "elec_MJ": 11.556,
+    "elec_MJ": 1.1556e1,
```

Comment:
the last 2 diff  `elec_MJ`are unrelated to the ior_correction. 
It's just that the number format was not respected before but the numbers stay the same